map/map_height_fitter/src/map_height_fitter.cpp

```diff
@@ -31,52 +37,73 @@ struct MapHeightFitter::Impl
   static constexpr char enable_partial_load[] = "enable_partial_load";
 
   explicit Impl(rclcpp::Node * node);
-  void on_map(const sensor_msgs::msg::PointCloud2::ConstSharedPtr msg);
+  void on_pcd_map(const sensor_msgs::msg::PointCloud2::ConstSharedPtr msg);
+  void on_vector_map(const autoware_auto_mapping_msgs::msg::HADMapBin::ConstSharedPtr msg);
   bool get_partial_point_cloud_map(const Point & point);
-  double get_ground_height(const tf2::Vector3 & point) const;
+  double get_ground_height(const Point & point);
```

Comment:
Can this function be const?

Reply:
It was a simple mistake. Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6340/commits/23e01b568d32634179f9a6fd01bf9776a26fe8dd